DfE.FindInformationAcademiesTrusts/Extensions/DateTimeExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using DfE.FindInformationAcademiesTrusts.Pages;
+
+namespace DfE.FindInformationAcademiesTrusts.Extensions;
+
+public static class DateTimeExtensions
+{
+    public static string ShowDateStringOrReplaceWithText(this DateTime? date, string replacementText = "No Data",
```

Comment:
The `replacementText` and `format` are never set. We should remove them as optional parameters and move the "No Data" into one of the constants files